.github/triage/jax_toolbox_triage/args.py

```diff
@@ -187,11 +187,25 @@ def parse_args(args=None) -> argparse.Namespace:
     parser.add_argument(
         "--container-runtime",
         default="docker",
-        help="Container runtime used, this can be either docker or pyxis.",
+        help="Container runtime used, can be docker, pyxis, or local.",
         type=lambda s: s.lower(),
     )
     args = parser.parse_args(args=args)
-    assert args.container_runtime in {"docker", "pyxis"}, args.container_runtime
+    assert args.container_runtime in {"docker", "pyxis", "local"}, args.container_runtime
+    
+    if args.container_runtime == "local":
+        assert (
+            args.passing_commits is not None and args.failing_commits is not None
+        ), "For local runtime, --passing-commits and --failing-commits must be provided."
+        assert (
+            args.container is None 
+            and args.start_date is None
+            and args.end_date is None
+            and args.passing_container is None
+            and args.failing_container is None
```

Comment:
`--container-mount` is also not going to do anything, plus `--container-url-template`